README.md

```diff
@@ -22,6 +22,9 @@ $ cp .env.example .env
 # fill in your own values in .env, then =>
 $ yarn
 $ yarn start
+
+# to populate the translations (optional)
+$ yarn lingui:build
 ```
```

Comment:
Given my PR: https://github.com/OlympusDAO/olympus-frontend/pull/719 this would make lingui:build mandatory because the `src/locales/translations` folder would be empty after cloning